how-tos/relpro.md

```diff
@@ -156,50 +156,87 @@ $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.
 $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.ini --branch-and-product-config=/home/cltbld/releasetasks/releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml --common-task-id=$TASK_TASKID_FROM_GRAPH1
 ```
 
+## 3. signoffs
```

Comment:
cool. I suppose, releasewarrior was intended to track "manual steps" as actions. Since there is no --sign-off action added, that's the part that may be a bit confusing. Perhaps add something about signoffs are not actually tracked in releasewarrior since their state is visible in balrog UI?